### PR TITLE
Order linters

### PR DESCRIPTION
**Pull request by Adirio**
Suggested in https://github.com/kubernetes-sigs/kubebuilder/pull/1310#discussion_r367616222

Tacked in #1309

/kind cleanup

### CODE REVIEW

**Comment by Adirio** [comment]
This PR needs #1291 and #1305 to be merged first

/hold

**Comment by Adirio** [comment]
/hold cancel

@camilamacedo86 @mengqiy Now that we added all the pending lints in PRs I think we should merge this before we include more of them, if we do.

**Comment by Adirio** [comment]
@camilamacedo86 Could you add `ok-to-test`?

@mengqiy Should we merge this?

File: scripts/verify.sh [hunk 1]
```diff
@@ -18,27 +18,27 @@ source ./common.sh
 
 header_text "running golangci-lint"
 cd .. # To go to the root of the project
-golangci-lint run --disable-all \
-    --deadline 5m \
-    --enable=govet \
-    --enable=gofmt \
-    --enable=misspell \
-    --enable=structcheck \
+# Keep the linters enable in separate and ordered lines to avoid duplicates
```
Comment (estroz) [inline scripts/verify.sh]:
```suggestion
# Keep the enabled linters in separate, ordered lines to avoid duplicates.
```

**Comment by Adirio** [comment]
@camilamacedo86 could you please re-add `lgtm` label, the small documentation changed was detected as a change, ty

**Pull request merged**